v22.1/ui-hot-ranges-page.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Hot Ranges Page
+summary: The Hot Ranges page provides details about ranges receiving a high number of reads or writes.
+toc: true
+docs_area: reference.db_console
+---
+
+The **Hot Ranges** page of the DB Console provides details about ranges receiving a high number of reads or writes. These are known as "hot ranges".
+
+When [optimizing](performance-best-practices-overview.html#hot-spots) or [troubleshooting](query-behavior-troubleshooting.html#single-hot-node) statement performance, this page can help you identify nodes, ranges, or tables that are experiencing hot spots.
+
+To view this page, [access the DB Console](ui-overview.html#db-console-access) and click **Hot Ranges** in the left navigation.
```

Comment:
In the other ui-* pages we refer to the sidebar as the left-hand navigation bar or left-hand navigation.